content/participants/ttjc@ashishjagrawal/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Ashish Jagrawal"
+type: "participant"
+name: "Ashish Kumar Jagrawal"
+portfolio: ""
+github-link: "https://github.com/Ashishjagrawal"
+linkedin-link: "https://linkedin.com/in/ashishkjagrawal"
+looking-for: "Job"
+description: "MERN Stack Developer"
+---
+
+Hi! I am Ashish Jagrawal. I am Currently  pursuing Bachelors of Technology in Computer Science from Arya College of Engineering and IT. I like eating Rajma Chawal.
+
+### introduction
+
+I always thought about how the websites and mobile apps work in my childhood and that interest took me to learn computer science and now I am a full stack developer. I am not much experienced but always willing to learn.
+
+## why are you taking this challenge?
+
+I need a job in July 2020.
+I want to create value by skills.
+
+## something interesting about you?
+
+I can talk about absolutely anything to anyone. I can  hack things to get my work done. I do not have biased political view. 
+
+### help I need?
+
+It would be great if someone make vaccine for this Corona Virus shit. 
```

Comment:
Hey @Ashishjagrawal ,
Why not you work for some coronavirus shit that would be great. This job challenge is not a joke that you are committing such nuisance here, this is the first step to any corporate world and if someone doesn't have work ethics before I don't think he will have after so better luck next time.

Reply:
Sorry, I thought this would be cool. And I am really serious about this challenge. I'll change it soon. (I didn't mean that)
Thanks

Reply:
Hey @gkanishk  ,
I have made changes. Please accept my pull request.

Thank you